### ISSUE TITLE
mvn jetty:run shows a not styled page

### ISSUE THREAD

**Issue by erickdeoliveiraleal** [primefaces/showcase-facelift#3]
![imagem 1](https://user-images.githubusercontent.com/3578953/41662245-0fcd02fa-7477-11e8-80b7-c08d42668dfd.png)

**Comment by melloware** [comment]
For those out there that use Windows I downloaded DART SASS for Windows.

https://github.com/sass/dart-sass/releases

And executed this command in the /css folder of the showcase.

 `sass.bat layout.scss layout.css --no-source-map`

**Comment by cnsgithub** [comment]
@melloware Should we add that to readme.md? Many user will face the very same problem.

**Comment by melloware** [comment]
Agreed I mentioned in on the Gitter Chat and @erickdeoliveiraleal said he submitted a PR for the documentation but I don't see any PR.

**Comment by melloware** [comment]
I will submit a PR.

**Comment by cnsgithub** [comment]
Ok, cool. Thanks.

Or would it be even better to integrate a maven plugin for the sass stuff? E.g. https://www.geodienstencentrum.nl/sass-maven-plugin/plugin-info.html

**Comment by melloware** [comment]
even better! Let me see if I can get that working.
